.drone/drone.jsonnet

```diff
@@ -310,6 +315,25 @@ local manifest(apps) = pipeline('manifest') {
 };
 
 [
+  pipeline('loki-build-image') {
+    workspace: {
+      base: '/src',
+      path: 'loki',
+    },
+    steps: [
+      {
+        name: 'build-image',
+        image: 'r.j3ss.co/img',
+        depends_on: ['clone'],
+        commands: [
+          'apk add --no-cache bash make',
+          'make BUILD_IN_CONTAINER=false build-image'
+        ],
+        when: condition('include').path('loki-build-image/**'),
+        // when: condition('include').tagMain + condition('include').path('loki-build-image/**'),
+      },
+    ],
+  },
```

Comment:
I'm not sure what the best way is to build the image. `make loki-build-image` depends on `img`. However, for the other images we use the Docker plugin for Drone CI. @dannykopping what would you use?

Reply:
I think you should chat with the original author @slim-bean; I have no idea